src/titiler/core/setup.py

```diff
@@ -12,7 +12,7 @@
     "numpy",
     "pydantic",
     "rasterio",
-    "rio-tiler@git+ssh://git@github.com/SenteraLLC/rio-tiler.git@3.1.5",
+    "rio-tiler@https://github.com/SenteraLLC/rio-tiler/archive/3.1.5.tar.gz",
```

Comment:
This `git+ssh` -> `https` protocol change helps me avoid installing git in the Dockerfile.